lib/openhab/dsl/items/builder.rb

```diff
@@ -511,11 +558,38 @@ def group(*groups)
         #     switch_item Bedroom_Light, thing: "mqtt:topic:bedroom-light", channel: :power
         #   end
         #
-        def channel(channel, config = {})
-          channel = channel.to_s
-          channel = "#{@thing}:#{channel}" if @thing && !channel.include?(":")
-          @channels << [channel, config]
+        # @example Multiple channels
```

Comment:
We should also add an example where multiple calls to #channel is done inside a block